test/presenters/return-versions/setup/additional-submission-options.presenter.test.js

```diff
@@ -54,24 +55,70 @@ describe('Return Versions Setup - Additional Submission Options presenter', () =
     })
   })
 
-  describe('the "additionalSubmissionOptions" property', () => {
-    describe('when the user has previously submitted additional submission options', () => {
+  describe('the "multipleUpload" property', () => {
+    describe('when a user has selected "multipleUpload" for additional options', () => {
```

Comment:
We seem to have dropped the word 'previously' from these tests. It was there to try and highlight where a user has previously visited this page and submitted an option, then we expect the presenter to replay it.

Happy to change it to something else if it's not making that clear. But I think just dropping it makes it unclear we are talking about selections a user previously made, instead of ones they have just made.